File: External/meegpipe/+mperl/+file/+spec/rootdir.m
function dir = rootdir()
% ROOTDIR - Returns a string representation of the root directory
%
%
% dir = filespec.rootdir;
%
% Where
%
% DIR is a string representation of the root directory
%
%
% See also: filespec

% (c) German Gomez-Herrero, german.gomezherrero@kasku.org

% Documentation: pkg_filespec.txt
% Description: Returns a string representation of the root directory


dir = perl('+mperl/+file/+spec/rootdir.pl');


end

File: External/meegpipe/+mperl/+file/+spec/rootdir.pl
#!/usr/bin/perl
# (c) German Gomez-Herrero, german.gomezherrero@kasku.org

# Description: Returns a string representation of the root directory
# Documentation: pkg_filespec.txt

use warnings;

# Make our Spec.pm visible
my $libLocation;
BEGIN {
    use Cwd 'abs_path';
    use File::Spec;
    my $name = abs_path($0);
    $name=~m/^.+\//; 
    $libLocation = abs_path(File::Spec->catdir($&, '../../lib'));
}

use lib $libLocation;

use File::Spec;

print File::Spec->rootdir();

